### PR TITLE
Bug in event handling for carbon_v1 pools

### PR DESCRIPTION
**Pull request by Lesigh-3100**
Issue described in: https://github.com/bancorprotocol/fastlane-bot/issues/419

### CODE REVIEW

**Review by barakman — CHANGES_REQUESTED** [review]

File: fastlane_bot/events/pools/carbon_v1.py [hunk 1]
```diff
@@ -79,24 +79,15 @@ def parse_event(
...
-            data["B_1"] = order1["B"]
+        data["y_0"] = order0[0]
+        data["z_0"] = order0[1]
+        data["A_0"] = order0[2]
+        data["B_0"] = order0[3]
+        data["y_1"] = order1[0]
+        data["z_1"] = order1[1]
+        data["A_1"] = order1[2]
+        data["B_1"] = order1[3]
+
```
[the remark this answers is not in the document]
Comment (barakman) [reply fastlane_bot/events/pools/carbon_v1.py:90]: Of course, I don't disagree that the previous code was bad by itself in several different aspects, such as:
- In function `parse_orders`, returning two different types
- In this function, deciding the course of action based on the input type
- In this function, assuming that either both inputs are of type `list` or both inputs are not of type `list`

[the remark this answers is not in the document]
Comment (Lesigh-3100) [reply fastlane_bot/events/pools/carbon_v1.py:96]: Updated this - could you please take another look?

File: fastlane_bot/events/pools/carbon_v1.py [hunk 2]
```diff
@@ -77,55 +77,25 @@ def parse_event(
...
-        return order0, order1
+        data["cid"] = event_args["args"].get("id")
+        data["y_0"] = order0[0]
+        data["z_0"] = order0[1]
+        data["A_0"] = order0[2]
+        data["B_0"] = order0[3]
+        data["y_1"] = order1[0]
+        data["z_1"] = order1[1]
+        data["A_1"] = order1[2]
+        data["B_1"] = order1[3]
```
Comment (barakman) [inline fastlane_bot/events/pools/carbon_v1.py:95]: Why not do everything in the same way that you are handling `cid`?

**Review by barakman — CHANGES_REQUESTED** [review]

**Comment by Lesigh-3100** [comment]
After testing I found that this code functions as-is. The issue is somewhere else in the code. Closing PR.

**Pull request closed without merging**